src/components/Shell/ShellAppBar.tsx

```diff
@@ -9,47 +9,66 @@ import MenuIcon from '@mui/icons-material/Menu'
 import LinkIcon from '@mui/icons-material/Link'
 
 import { drawerWidth } from './Drawer'
+import { peerListWidth } from './PeerList'
 
 interface AppBarProps extends MuiAppBarProps {
-  open?: boolean
+  isDrawerOpen?: boolean
+  isPeerListOpen?: boolean
 }
 
 export const AppBar = styled(MuiAppBar, {
-  shouldForwardProp: prop => prop !== 'open',
-})<AppBarProps>(({ theme, open }) => ({
+  shouldForwardProp: prop =>
+    prop !== 'isDrawerOpen' && prop !== 'isPeerListOpen',
+})<AppBarProps>(({ theme, isDrawerOpen, isPeerListOpen }) => ({
   transition: theme.transitions.create(['margin', 'width'], {
     easing: theme.transitions.easing.sharp,
     duration: theme.transitions.duration.leavingScreen,
   }),
-  ...(open && {
+  ...(isDrawerOpen && {
     width: `calc(100% - ${drawerWidth}px)`,
     marginLeft: `${drawerWidth}px`,
     transition: theme.transitions.create(['margin', 'width'], {
       easing: theme.transitions.easing.easeOut,
       duration: theme.transitions.duration.enteringScreen,
     }),
   }),
+  ...(isPeerListOpen && {
+    width: `calc(100% - ${peerListWidth}px)`,
+    marginRight: `${peerListWidth}px`,
+    transition: theme.transitions.create(['margin', 'width'], {
+      easing: theme.transitions.easing.easeOut,
+      duration: theme.transitions.duration.enteringScreen,
+    }),
+  }),
 }))
 
 interface ShellAppBarProps {
   doShowPeers: boolean
   handleDrawerOpen: () => void
   handleLinkButtonClick: () => Promise<void>
   isDrawerOpen: boolean
+  isPeerListOpen: boolean
   numberOfPeers: number
   title: string
+  handlePeerListOpen: () => void
```

Comment:
We should [use this article](https://jaketrent.com/post/naming-event-handlers-react/) as a guide for event handler naming in Chitchatter. Specifically, event handler component props should use the `on{Subject}{Event}` format. So, this should be:

```suggestion
  onPeerListOpen: () => void
```

Reply:
thx :)